source/atlas-cli-changelog.txt

```diff
@@ -21,7 +21,12 @@
 
 Released 5 July 2023
 
-- Adds the ability to set up Microsoft Teams alert notifications.
+.. important::
+   
+   {+atlas-cli+} won't be available on 4.2 linux (Debian, Ubuntu and rhel) repositories after this release.
```

Comment:
```suggestion
   This is the last {+atlas-cli+} release to be available on the mongodb server 4.2 Linux repositories for Debian, Ubuntu and RHEL.
```
Sorry I wasn't a bit clearer, we publish new updates of the CLI to the server repos, given 4.2 is EOL will be stopping that but older version will still be avialble up to 1.9.1, also I imagine theres a reference like {+atlas-cli+} for the server but I don't know it